munki_rebrand.py

```diff
@@ -326,7 +326,13 @@ def convert_to_icns(png, output_dir, actool=""):
     # Munki 3.6+ has an Assets.car which is compiled from the Assets.xcassets
     # to provide the AppIcon
     if actool:
-        shutil.copytree("./Assets.xcassets/", xcassets, dirs_exist_ok=True)
+        # Use context of the location of munki_rebrand.py to find the Assets.xcassets directory.
+        rebrand_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), os.pardir))
```

Comment:
Going to suggest that `os.path.dirname(os.path.abspath(__file__))` is easier to grok here